src/Accelerators/NNPA/Conversion/ONNXToZHigh/ONNXToZHigh.td

```diff
@@ -36,6 +36,10 @@ class HasRankOf<int rank> : Constraint<
   CPred<"$0.getType().isa<ShapedType>() && $0.getType().cast<ShapedType>().getRank() == " # rank>
 >;
 
+def IsBiasNoneOr1D : Constraint<
+  CPred<"(($_self).getType().isa<NoneType>()) || (($_self).getType().isa<ShapedType>() && ($_self).getType().cast<ShapedType>().getRank() == 1)">
```

Comment:
This line is too long, please split it into multiple short lines, e.g.
```
"($_self.getType().isa<NoneType>() || "
"  ($_self.getType().isa<ShapedType>() && "
"   $_self.getType().cast<ShapedType>().hasRank() &&"
"   ($_self.getType().cast<ShapedType>().getRank() == 1))"
```

There is a bug that we need to check `$_self.getType().cast<ShapedType>().hasRank()` before `getRank`. This PR #1456 addressed that issue. Could you fix the same issue in this file and possibly other files? Thanks!

Reply:
Done. Thanks!
I checked NNPA code and fix it to use `hasRank()`.